examples/step-57/doc/results.dox

```diff
@@ -0,0 +1,429 @@
+    <th>3.35933e-11 </th>
+    <th>3         </th>
+    <th>5.86734e-11 </th>
+    <th>2         </th>
+  </tr>
+  <tr>
+    <th>4         </th>
+    <th>9.26748e-4  </th>
+    <th>4         </th>
+    <th>2.75537e-14  </th>
+    <th>4         </th>
+    <th>1.39986e-14 </th>
+    <th>5         </th>
+    <th>2.18864e-14 </th>
+    <th>5         </th>
+    <th>3.38787e-14 </th>
+    <th>5         </th>
+  </tr>
+  <tr>
+    <th>5          </th>
+    <th>1.34601e-5</th>
+    <th>4          </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+  </tr>
+  <tr>
+    <th>6         </th>
+    <th>2.5235e-8    </th>
+    <th>5    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+  </tr>
+  <tr>
+    <th>7         </th>
+    <th>1.38899e-12    </th>
+    <th>4    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+  </tr>
+  <tr>
+    <th>8           </th>
+    <th>4.68224e-15 </th>
+    <th>4           </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+  </tr>
+</table>
+
+The following figures show the sequence of the generated grids. For the case
+of Re=400, the initial guess is obtained by solving Stokes on an $8 \times 8$
+mesh, and the mesh is refined adaptively. Between meshes, the solution from
+the coarse mesh is interpolated to the fine mesh to be used as an initial guess.
+
+<TABLE ALIGN="center">
+  <tr>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re400_Mesh0.png" width="232px" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re400_Mesh1.png" width="232px" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re400_Mesh2.png" width="232px" alt="">
+    </td>
+  </tr>
+  <tr>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re400_Mesh3.png" width="232px" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re400_Mesh4.png" width="232px" alt="">
+    </td>
+  </tr>
+</table>
+
+This picture is the graphical result of lid-driven cavity with Re=400.
+<img src="https://www.dealii.org/images/steps/developer/step-57.Re400_Streamline.png" alt="">
+
+Then the solution by Newton's iteration is compared with a reference solution
+from [4].
```

Comment:
It would also be worthwhile saying what the figure actually shows. Mention that the data shown here is included in files named XXYY in the source directory.